Added new test, CPU-OFFLINE-NETWORK-WORKLOAD

juhlee-microsoft opened the pull request:
Added new TC definition in the xml file.
    Modified the PS1 script managing network workload.
    Moving the reset codes to before rebooting step

**TEST RESULT**

[LISAv2 Test Results Summary]
Test Run On           : 07/10/2020 01:05:23
ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
Initial Kernel Version: 5.3.0-1032-azure
Final Kernel Version  : 5.8.0-rc1+
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:28

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes)
-------------------------------------------------------------------------------------------------------------------------------------------
    1 CORE                 CPU-OFFLINE-NETWORK-WORKLOAD                                                      PASS                25.59


[LISAv2 Test Results Summary]
Test Run On           : 07/10/2020 02:32:51
ARM Image Under Test  : Canonical : UbuntuServer : 16.04-LTS : latest
Initial Kernel Version: 4.15.0-1091-azure
Final Kernel Version  : 5.8.0-rc1+
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:25

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes)
-------------------------------------------------------------------------------------------------------------------------------------------
    1 CORE                 CPU-OFFLINE-NETWORK-WORKLOAD                                                      PASS                 22.4

juhlee-microsoft committed:
Added new test, CPU-OFFLINE-NETWORK-WORKLOAD
    Added new TC definition in the xml file.
    Modified the PS1 script managing network workload.
    Moving the reset codes to before rebooting step
    Changed to iperf3
    Non Canonical distro need extra disk for kernel compilation
    Kernel compilation step update for rhel/centos8
    Adjust for CentOS8
    Moving the reset codes to before rebooting step

juhlee-microsoft committed:
Added new test, CPU-OFFLINE-NETWORK-WORKLOAD
    Added new TC definition in the xml file.
    Modified the PS1 script managing network workload.
    Moving the reset codes to before rebooting step

juhlee-microsoft committed:
Added new test, CPU-OFFLINE-NETWORK-WORKLOAD

Added new TC definition in the xml file.
Modified the PS1 script accomodating network workload.

juhlee-microsoft committed:
Update CPU-Offline.ps1

Fxied the error

juhlee-microsoft committed:
Update CPU-Offline.ps1

juhlee-microsoft committed:
Update CPU-Offline.ps1

juhlee-microsoft committed:
Update CPU-Offline.ps1

juhlee-microsoft committed:
Update CPU-Offline.ps1

Moving the reset codes to before rebooting step

juhlee-microsoft committed:
Update CPU-Offline.ps1

juhlee-microsoft committed:
Update CPU-Offline.ps1

juhlee-microsoft committed:
Update CPU-Offline.ps1

lubaihua33 reviewed (commented):
> There is no iperf package installation in RHEL 8.x, CentOS 8.x and SLES 15 SP1. SLES should add the depreciate net util repo, but not sure if there is better solution.

How do you like ntttcp? If we choose to download and compile code, ntttcp may be a good choise to test network.

juhlee-microsoft committed:
Update CPU-Offline.ps1

Changed to iperf3

juhlee-microsoft committed:
Merge branch 'juhlee/cpu_network_workload' of https://github.com/LIS/LISAv2 into juhlee/cpu_network_workload

juhlee-microsoft committed:
Update CPU-Offline.ps1

Non Canonical distro need extra disk for kernel compilation

juhlee-microsoft committed:
Update CPU-Offline.ps1

juhlee-microsoft committed:
Update CPUOfflineKernelBuild.sh

Kernel compilation step update for rhel/centos8

juhlee-microsoft committed:
Adjust for CentOS8

juhlee-microsoft committed:
Merge branch 'juhlee/cpu_network_workload' of https://github.com/LIS/LISAv2 into juhlee/cpu_network_workload

juhlee-microsoft committed:
Update utils.sh

SUSE messed up iperf and iperf3. iperf3 entry point removed in the repo, but combined to iperf.

juhlee-microsoft committed:
Merge branch 'juhlee/cpu_network_workload' of https://github.com/LIS/LISAv2 into juhlee/cpu_network_workload

juhlee-microsoft committed:
Update utils.sh

Added iperf soft link

juhlee-microsoft committed:
Fixed with the PR comments

1. Fixed the timout value
2. Re-wording
3. Change to the number of cpu in the VM

juhlee-microsoft committed:
Update CPU-Offline.ps1

juhlee-microsoft committed:
Update CPU-Offline.ps1

juhlee-microsoft committed:
Update CPU-Offline.ps1

juhlee-microsoft committed:
Update CPUOfflineKernelBuild.sh

Change the code pick up the only new vmlinuz image file

juhlee-microsoft committed:
Update CPU-Offline.ps1

juhlee-microsoft committed:
The default VM size change to Standard_D32as_v4

juhlee-microsoft committed:
Update CPU-Offline.ps1

Fixed the error.

juhlee-microsoft committed:
Merge branch 'master' into juhlee/cpu_network_workload

harz566 committed:
fix regression for -UseExistingRG (#905)

harz566 committed:
update ReadMe for refactor (#906)

juhlee-microsoft committed:
Merge branch 'master' into juhlee/cpu_network_workload

juhlee-microsoft committed:
Update FunctionalTests.xml

Fixed from rebase failure

juhlee-microsoft committed:
Revert "update ReadMe for refactor (#906)"

This reverts commit 22aab7dcd477f3fbd6b562927a239842c9388797.
